server/members/views.py

```diff
@@ -3,47 +3,48 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from rest_framework.permissions import IsAuthenticated
-from .models import Member
-from .serializers import MemberSerializer
+from .models import User
+from .serializers import UserSerializer
 from .permissions import IsAuthenticatedOrReadOnlyWithAPIKey
 from django.shortcuts import get_object_or_404
+from django.contrib.auth.models import Group
 
 class MembersList(generics.ListCreateAPIView):
-    queryset = Member.objects.all()
-    serializer_class = MemberSerializer
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
 
 class MemberRetrieveUpdateDestroy(generics.RetrieveUpdateDestroyAPIView):
-    queryset = Member.objects.all()
-    serializer_class = MemberSerializer
+    queryset = User.objects.all()
+    serializer_class = UserSerializer
 
     def get(self, request, member_id):
```

Comment:
might be able to shorten with `get_object_or_404`? although is this different at all from the default behavior of the base api view? (I know this wasn't from this PR, but since we are refactoring...)